components_new/query_handler/queryHandler.py

```
@@ -0,0 +1,41 @@
+import json
+from queryFunctions import *
```

Ruff (0.8.2)

`from queryFunctions import *` used; unable to detect undefined names

(F403)